QEfficient/cloud/execute.py

```diff
@@ -18,29 +18,38 @@
 
 def main(
     model_name: str,
-    prompt: str,
     qpc_path: str,
-    devices: List[int],
+    device_group: List[int],
+    prompt: str = None,
+    prompts_txt_file_path: str = None,
     cache_dir: str = Constants.CACHE_DIR,
     hf_token: str = None,
 ):
     """
     APi() to run the Model on Cloud AI 100 Platform.
     ---------
-    :param model_name: str. Hugging Face Model Card name, Example: [gpt2]
-    :prompt: str. Sample prompt for the model text generation
+    :param model_name: str. Hugging Face Model Card name, Example: "gpt2"
     :qpc_path: str.  Path to the generated binary after compilation.
-    :devices: List[int]. Device Ids to be used for compilation. if devices > 1. Multiple Card setup is enabled.
+    :device_group: List[int]. Device Ids to be used for compilation. if len(device_group) > 1. Multiple Card setup is enabled.
+    :prompt: str. Sample prompt for the model text generation
+    :prompts_txt_file_path: str. Path to txt file for multiple input prompts (in case of batch size > 1)
     """
+
     if hf_token is not None:
         login(hf_token)
+
     # Download tokenizer along with model if it doesn't exist
-    model_hf_path = hf_download(repo_id=model_name, cache_dir=cache_dir, allow_patterns=["*.json"])
-    tokenizer = AutoTokenizer.from_pretrained(
-        model_hf_path, use_cache=True, padding_side="left", trust_remote_code=True
-    )
+    model_hf_path = hf_download(repo_id=model_name, cache_dir=cache_dir, allow_patterns=["*.json", "*.py"])
```

Comment:
Add `*token*` in `allow_patterns`, current code won't download `tokenizer.model`

Reply:
done